Makefile

```diff
@@ -140,8 +140,44 @@ $(YUI): $(NODE_MODULES)
 $(REACT_ASSETS): $(NODE_MODULES)
 	cp $(NODE_MODULES)/react/dist/react-with-addons.js $(BUILT_JS_ASSETS)/react-with-addons.js
 	cp $(NODE_MODULES)/react/dist/react-with-addons.min.js $(BUILT_JS_ASSETS)/react-with-addons.min.js
+	sed	-i -e '/var NoopInternalComponent = function (element) {/,/^};/c\
```

Comment:
Nice work. It's probably worth adding a big comment here about what this is for and when we can remove it.